hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -172,3 +173,43 @@ def _check_date_state_one(self):
                 _("You can't create duplicate public holiday per date %s.") % self.date
             )
         return True
+
+    def _prepare_holidays_meeting_values(self):
+        self.ensure_one()
+        categ_id = self.env.ref("hr_holidays_public.event_type_holiday", False)
+        meeting_values = {
+            "name": (
+                "{} ({})".format(self.name, self.year_id.country_id.name)
+                if self.year_id.country_id
+                else self.name
+            ),
+            "description": ", ".join(self.state_ids.mapped("name")),
+            "categ_ids": [(6, 0, categ_id.ids if categ_id else [])],
+            "start": self.date,
+            "stop": self.date,
+            "allday": True,
+            "partner_ids": False,
+            "user_id": SUPERUSER_ID,
```

Comment:
```suggestion
            "user_id": self.env.uid,
```

??

Reply:
The person who creates the public holiday is not relevant here. I think it's better to use __system__ for this